<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newItem</class>
 <widget class="QDialog" name="newItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Item</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <activeon>:/paborito Logo.png</activeon>
   </iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Kozuka Gothic Pr6N B</family>
           <pointsize>8</pointsize>
           <underline>true</underline>
          </font>
         </property>
         <property name="text">
          <string>Price</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Kozuka Gothic Pr6N B</family>
           <pointsize>8</pointsize>
           <underline>true</underline>
          </font>
         </property>
         <property name="text">
          <string>Description</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Kozuka Gothic Pr6N B</family>
           <pointsize>8</pointsize>
           <underline>true</underline>
          </font>
         </property>
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <family>Kozuka Gothic Pr6N B</family>
           <pointsize>10</pointsize>
           <underline>true</underline>
          </font>
         </property>
         <property name="text">
          <string>Category</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QLineEdit" name="lineEdit_FoodName">
         <property name="font">
          <font>
           <family>Kozuka Gothic Pr6N B</family>
           <pointsize>8</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLineEdit" name="lineEdit_Description">
         <property name="font">
          <font>
           <family>Kozuka Gothic Pr6N B</family>
           <pointsize>8</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QComboBox" name="comboBox_Categ">
         <property name="font">
          <font>
           <family>Kozuka Gothic Pr6N B</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Rice Bowls</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Finger Foods</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Pasta</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Pizza</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Specialty</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Cold Drinks</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Hot Drinks</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Others</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_Price">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="maximum">
          <double>999999999.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>Kozuka Gothic Pr6N B</family>
         <pointsize>8</pointsize>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Set Image</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_SetImage">
       <property name="font">
        <font>
         <family>Kozuka Gothic Pr6N B</family>
         <pointsize>8</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton_SetImage">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Kozuka Gothic Pr6N B</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 255, 255);</string>
       </property>
       <property name="text">
        <string>/</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_Add">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Kozuka Gothic Pr6N B</family>
         <pointsize>13</pointsize>
         <weight>50</weight>
         <bold>false</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 85, 255);</string>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_Cancel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Kozuka Gothic Pr6N B</family>
         <pointsize>13</pointsize>
         <weight>50</weight>
         <bold>false</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 170, 0);</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Kozuka Gothic Pr6N B</family>
      </font>
     </property>
     <property name="title">
      <string>Pizza Sizes</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_11">
      <item row="0" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>15</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>16</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <layout class="QGridLayout" name="gridLayout_10">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Regular</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QDoubleSpinBox" name="doubleSpinBox">
            <property name="maximum">
             <double>999999999.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="0" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Large</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
            <property name="maximum">
             <double>999999999.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="2">
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="0" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Jumbo</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
            <property name="maximum">
             <double>999999999.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBox_Categ</tabstop>
  <tabstop>lineEdit_FoodName</tabstop>
  <tabstop>lineEdit_Description</tabstop>
  <tabstop>doubleSpinBox_Price</tabstop>
  <tabstop>doubleSpinBox</tabstop>
  <tabstop>doubleSpinBox_2</tabstop>
  <tabstop>doubleSpinBox_3</tabstop>
  <tabstop>lineEdit_SetImage</tabstop>
  <tabstop>pushButton_SetImage</tabstop>
  <tabstop>pushButton_Add</tabstop>
  <tabstop>pushButton_Cancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
